### PR TITLE
 Adding numer of patients booked in appointment status

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/REPORTING/load_aggregate_Appointments.sql [hunk 1]
```diff
@@ -1,37 +1,75 @@
...
+  NumOfPatients, 
+  DATIMAgeGroup, 
+  summary.AsOfDateKey, 
+  NumBooked, 
+  cast(
+    getdate() as date
+  ) as LoadDate into Reporting.dbo.AggregateAppointments 
+from 
+  Summary 
+  left join Bookings on Bookings.FacilityKey = Summary.FacilityKey
```
[the remark this answers is not in the document]
Comment (Marymary-dev) [reply Scripts/REPORTING/load_aggregate_Appointments.sql]: Done

File: Scripts/REPORTING/load_aggregate_Appointments.sql [hunk 2]
```diff
@@ -1,37 +1,94 @@
...
+      partner.PartnerName, 
+      Partner.PartnerKey, 
+      agency.AgencyName, 
+      agency.AgencyKey, 
+      patient.Gender, 
+      AppointmentStatus, 
+      apt.AsOfDateKey, 
+      age_group.DATIMAgeGroup
+  ) 
+Select
```
[the remark this answers is not in the document]
Comment (Marymary-dev) [reply Scripts/REPORTING/load_aggregate_Appointments.sql:68]: Done

File: Scripts/REPORTING/load_aggregate_Appointments.sql [hunk 3]
```diff
@@ -1,37 +1,91 @@
...
+  Summary.SubCounty, 
+  Summary.County, 
+  Summary.PartnerName, 
+  Summary.AgencyName, 
+  Summary.Gender, 
+  AppointmentStatus, 
+  NumOfPatients, 
+  Bookings.NumBooked, 
+  Summary.DATIMAgeGroup, 
+  Summary.AsOfDateKey,
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_Appointments.sql:80]: @Marymary-dev just a small one we can have this as a date to enable plotting time series on the BI tools.

**Review by nobert-mumo — CHANGES_REQUESTED** [review]

File: Scripts/REPORTING/load_aggregate_Appointments.sql (hunk 3 above)
Comment (Marymary-dev) [reply Scripts/REPORTING/load_aggregate_Appointments.sql]: Done

**Review by nobert-mumo — APPROVED** [review]

**Comment by nobert-mumo** [comment]
@Marymary-dev LGTM, didn't check the actual data, this looks similar to the one on HTS Uptake + TX New model that had a bug. Hopefully, it's taken care of.

**Comment by nobert-mumo** [comment]
@Marymary-dev, pushed the changes, you can have a look.

**Pull request merged**